config/routes.rb

```diff
@@ -8,9 +8,18 @@
   # Example of regular route:
   #   get 'products/:id' => 'catalog#view'
 
+  resources :donors
+  resources :donations do
```

Comment:
What should be the root route in that case?
Instead of `root to: "donations#index"`